packages/fxa-settings/src/pages/Signin/SigninUnblock/container.tsx

```diff
@@ -78,10 +77,17 @@ const SigninUnblockContainer = ({
     signInOptions?: SignInOptions
   ) => {
     const service = integration.getService();
+    const clientId = integration.getClientId();
+
+    const isBrowserClient = service === 'sync' || service === 'relay';
```

Comment:
Just a heads up that in #17869, I added a [new enum for this](https://github.com/mozilla/fxa/pull/17869/files#diff-18cf62e8d5e61cb60bc712194a743a518f96d5a897e40a91628fdefb6ffce357R42), but unless that's merged before this we can update it later ofc. We might even want to add a helper function for the check to remove some redundancy.